src/main/java/ch/njol/skript/expressions/ExprShuffledList.java

```diff
@@ -62,14 +62,14 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	
 	@Override
 	@Nullable
-	protected Object[] get(Event e) {

```

Comment:
This should be `e`, @bensku even specifically changed it from `event` to `e` in one of my commits